spec/rails_helper.rb

```diff
@@ -8,6 +8,6 @@
 Dir[Rails.root.join("spec", "support", "**", "*.rb")].sort.each { |f| require f }
 
 RSpec.configure do |config|
-  config.use_transactional_fixtures = false
+  config.use_transactional_fixtures = true
```

Comment:
What is the reason for this change?

Reply:
"Sign In" spec fails when use_transactional_fixtures is FALSE, for now I cant' say actually why it happens.